test/phoenix_test/playwright_test.exs

```diff
@@ -23,11 +23,6 @@ defmodule PhoenixTest.PlaywrightTest do
   end
 
   describe "screenshot/3" do
-    setup do
-      File.rm_rf("screenshots")
-      :ok
-    end
-
```

Comment:
This will break the tests below that assert on `File.exists?/1`, since a dev can screw up the screenshot functionality, but if they ran the test previously on a working version, the test will still pass.

Reply:
True.
To avoid issues with `async` tests and deleting folders I'll just add a timestamp to the test file names.

Reply:
And for the CI integration test (auto-screenshot verified via GH workflow step) we're fine, because in CI that folder is empty.